File: quark/test/emit/int-methods/py/int_methods.py
from quark_runtime import *

class Test(object):
    def _init(self): pass
    def __init__(self): self._init()

    def test(self):
        x = 1;
        y = 2;
        z = (((x) + (y)) - (3)) * (4);
        _println(z);
        four = (2) + (2);
        _println(four);



def main():
    (Test()).test();


if __name__ == "__main__":
    main()

File: quark/test/emit/int-methods/js/int-methods.js
var _qrt = require("quark_runtime.js");

// CLASS Test
function Test() {
    this.__init_fields__();
}
exports.Test = Test;

function Test__init_fields__() {}
Test.prototype.__init_fields__ = Test__init_fields__;

function Test_test() {
    var x = 1;
    var y = 2;
    var z = (((x) + (y)) - (3)) * (4);
    _qrt.print(z);
    var four = (2) + (2);
    _qrt.print(four);
}
Test.prototype.test = Test_test;

function main() {
    (new Test()).test();
}
exports.main = main;

main();
